array.c

```diff
@@ -3473,25 +3462,21 @@ rb_ary_sort_by_bang(VALUE ary)
 
 /*
  *  call-seq:
- *    array.map {|element| ... } -> new_array
- *    array.map -> new_enumerator
```

Comment:
In the Ruby code I've seen, `map` is more popular than `collect`, so I would make that the primary method, use it in the examples, and say that `collect` is an alias for `map`.  Same goes for `map!`. But maybe that is just my personal preference.

Reply:
Personally, I agree.  I was just looking at C function names:  rb_ary_select, rb_ary_select_bang.  If no one objects, I'll change them.